src/it/multi-modules-aggregate/maven_checks.xml

```diff
@@ -67,7 +67,7 @@ under the License.
         <property name="tabWidth" value="4"/>
 
         <!-- required for SuppressWarningsFilter (and other Suppress* rules not used here) -->
-        <!-- see http://checkstyle.sourceforge.net/config_annotation.html#SuppressWarningsHolder -->
+        <!-- see https://checkstyle.sourceforge.io/config_annotation.html#SuppressWarningsHolder -->
```

Comment:
If this is the main checkstyle's documentation, then even this URL is outdated. We recommend everyone to use our `.org` domain. We are leaving the sourceforge domain up as a backup.
The URL for this would be: https://checkstyle.org/config_annotation.html#SuppressWarningsHolder

Reply:
Good catch, it's already fixed :innocent: